src/local-history/local-history-manager.ts

```diff
@@ -410,4 +395,11 @@ export class LocalHistoryManager {
         }
         return true;
     }
+
+    private checksum(str: string, algorithm: string = 'md5') {
+        return crypto
+            .createHash(algorithm)
+            .update(str, 'utf8')
+            .digest('hex');
+    }
 }
```

Comment:
should we revert this? no modification here

Reply:
I put it in the class, there are no outside callers so I didn't really understand why it was a `function`.